packages/tree/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@jupyter-notebook/tree",
+  "version": "7.0.0-alpha.5",
+  "description": "Jupyter Notebook - Tree ",
```

Trailing whitespace spotted:

```suggestion
  "description": "Jupyter Notebook - Tree",
```